spaces/aliases.json

```
@@ -48,5 +48,7 @@
   "yaxis": "yaxis.eth",
   "spaceswap": "spaceswap.eth",
   "descience.eth": "people-dao.eth",
-  "dforce": "dforcenet.eth"
+  "dforce": "dforcenet.eth",
+  "rari": "fuse.eth",
+  "fuse": "fuse.eth"
```

There isn't any "fuse" space, adding "fuse" is not necessary here
```suggestion
  "rari": "fuse.eth"
```